go/storage/client/init.go

```diff
@@ -54,6 +54,7 @@ func New(
 		namespace,
 		scheduler.KindStorage,
 		committee.WithAutomaticEpochTransitions(),
+		committee.WithFilter(committee.IgnoreNodeFilter(ident.NodeSigner.Public())),
```

Comment:
Actually, this should probably not be done in general (e.g., a compute node that is also a storage node may need to commit to self), only specifically for the storage node worker connecting to other nodes. You could make `New` take `committeeOpts ...committee.WatcherOption` and only enable it in the storage node worker.